draft-ietf-webpush-protocol.xml

```diff
@@ -504,6 +508,13 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
           server.
         </t>
         <t>
+         An application server MAY omit the receipt subscription if it is

```

Comment:
Receipt receiving may require (optional) VAPID, just like send. Monitoring or sending on behalf of other senders would be impossible ( or require sharing private keys )